catalogue/webapp/pages/works.tsx

```diff
@@ -287,13 +287,20 @@ export const getServerSideProps: GetServerSideProps<
   const globalContextData = getGlobalContextData(context);
   const parsedParams = parseUrlParams(context.query);
   const params = WorksRoute.fromQuery(parsedParams);
+  const { searchMoreFilters } = globalContextData.toggles;
+  const isImageSearch = params.search === 'images';
```

Comment:
is this needed?

Reply:
i dont this a merge conflict ,,, nice one .. ill remove it